client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -23,6 +25,15 @@ interface SubmitPitchModalProps extends ModalProps {
   callback(): void;
 }
 
+const notify = (): string =>
+  toast.error('Please fill out all the fields before submitting!', {
+    position: 'bottom-right',
+  });
+const interestNotify = (): string =>
+  toast.error("You can't select more than 4 interests", {
+    position: 'bottom-right',
+  });
```

Comment:
We may want to wrap this `toast.error` call in another method and put that method in a helper file somewhere. This will allow us to make sure all toast positions and styles are consistent whenever a toast in used in any file.

Reply:
Since we only use the toast in this file now, it's probably not necessary to do this yet.